simulators/nav/src/components/hotkeys.md

```diff
@@ -0,0 +1,36 @@
+# <img src="../static/mrover.png" alt="MRover Logo" width="30"/> MRover Navigation Simulator Hotkeys Guide
+
+| Hotkey | Description | File |
+| :----- | :---------- | :--- |
+|| <div align="center">_LCM Messages_</div> ||
+| `shift+enter` | Turn the rover on/off. | `HotKeys.vue` |
+|| <div align="center">_Debug Tools_</div> ||
+| `shift+space` | Play/pause the simulator. | `HotKeys.vue` |
+| `shift+alt+space` | When the rover is paused, take a "step" by applying a single joystick command then pausing again. | `HotKeys.vue` |
+| `shift+up` | Apply a single drive forward joystick command. | `HotKeys.vue` |
+| `shift+down` | Apply a single drive backward joystick command. | `HotKeys.vue` |
+| `shift+left` | Apply a single turn left joystick command. | `HotKeys.vue` |
+| `shift+right` | Apply a single turn right joystick command. | `HotKeys.vue` |
+| `shift+r` | Turn off the rover and move it to the starting location. | `HotKeys.vue` |
```

Comment:
this shift+r command is implemented in DebugTools.vue, not HotKeys.vue

Reply:
Good catch on this one too!